juniper_codegen/src/util/mod.rs

```diff
@@ -680,6 +706,9 @@ pub struct GraphQLTypeDefinitionField {
     pub is_async: bool,
     pub default: Option<TokenStream>,
     pub span: Span,
+
+    #[cfg(feature = "tracing")]
+    pub tracing: Option<tracing::Attr>,
```

Comment:
@ArsileLuci also, as `#[tracing]` represent a separate attribute, it woud be nice to not embed it here, but rather parse separately and use directly.

Reply:
@tyranron 

> parse separately and use directly.

You mean instead of expanding it within `#[graphql_object]`/`#[graphql_interface]` etc, we could just expand it separately for each field?